src/pages/DeliveryModeScreen/DeliveryModeScreen.tsx

```diff
@@ -0,0 +1,56 @@
+import { useContext } from 'react';
+import { Form } from 'react-final-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import { Select } from 'mui-rff';
+import FormScreens from 'constants/formScreens';
+import WizardFormContext, {
+  initialFormValues,
+} from 'contexts/WizardFormContext';
+import { validateIsRequired } from 'utils/validation';
+
+const DELIVERY_MODE_OPTIONS = {
```

Comment:
`DELIVERY_MODE_OPTIONS` should be an array of objects that include `label` and `value` from `DELIVERY_MODE_VALUES`. They should be mapped inside `Select` or use prop `data` (preferrably)